src/components/accordion/accordion.scss

```diff
@@ -5,13 +5,17 @@
 @use '../../scss/mixins/typography';

```

Comment:
Import the global.scss file here too (and in every component). That will give you `* { box-sizing: border-box }` on everything.